Classes/Networking/TMURLEncoding.m

```diff
@@ -80,10 +80,10 @@ + (void)encodeObject:(id)object withKey:(NSString *)key andSubKey:(NSString *)su
         }
     }
     else if ([object isKindOfClass:[NSArray class]]) {
-        for (NSString *arrayObject in (NSArray *)object) {
-            NSString *arrayKey = [[NSString alloc] initWithFormat:@"%@[]", objectKey];
+        [(NSArray *)object enumerateObjectsUsingBlock:^(NSString *arrayObject, NSUInteger idx, BOOL * stop) {
+            NSString *arrayKey = [[NSString alloc] initWithFormat:@"%@[%lu]", objectKey,  (unsigned long)idx];
```

Comment:
Nit: you can get rid of the alloc init using the class method instead

Reply:
Done.